### ISSUE TITLE
E2E setup teaching tooltip

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhome#498]
### Steps to reproduce the bug

Description
Open E2e Setup flow there should be a teaching tip inform the user all the changes will apply in the end of the set up flow

**Comment by dhoehna** [comment]
Tooltip exists that says "Your changes won't be applied until the end."
